### ISSUE TITLE
"Invalid or missing banlist.dat" message appears every start

### ISSUE THREAD

**Issue by laanwj** [bitcoin/bitcoin#7452]
It looks like banlist.dat is only actually created if nodes are banned. Otherwise, this will appear on every start:
```
2016-02-01 09:55:14 ERROR: Read: Failed to open file /home//.bitcoin/banlist.dat
2016-02-01 09:55:14 Invalid or missing banlist.dat; recreating
```
- The `ERROR:` makes it look like a serious issue
- It's probably best to just create the file there. But at the very least, the log message shouldn't promise to do something it isn't going to do

**Comment by kirkalx** [comment]
Partially resolved by PR #7458, but perhaps the issue could be left open until I (or someone else) can stop the ERROR message as well.

**Comment by laanwj** [comment]
Closing this - the primary problem has been solved in #7458, I don't think the one-time error is that bad.
